crates/emitter/src/emitter.rs

```diff
@@ -109,6 +110,11 @@ impl BytecodeOffset {
     fn new(offset: usize) -> Self {
         Self { offset }
     }
+
+    pub fn end(self, emit: &InstructionWriter) -> usize {
```

Comment:
can you add comment that explicitly says this is the offset of the end of opcode at this offset?
(or maybe add some more descriptive name)
it's not clear what chunk it refers when saying `end`.